monitoring: bare bones prometheus config and rules.

mberhault opened the pull request:
This is meant to be used in the docs and is a working config/rules for a
local node (with a single line to change for secure/insecure).

Work torwards https://github.com/cockroachdb/docs/issues/469

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/12288)
<!-- Reviewable:end -->

Commit:
monitoring: bare bones prometheus config and rules.

This is meant to be used in the docs and is a working config/rules for a
local node (with a single line to change for secure/insecure).

Work torwards https://github.com/cockroachdb/docs/issues/469

jseldess commented on monitoring/prometheus.yml:
```diff
@@ -0,0 +1,27 @@
+    static_configs:
+    - targets: ['localhost:8080']
+      labels:
+        cluster: 'cockroach-test'
```
Let's maybe rename this to something like 'my-cockroachdb-cluster"?

jseldess reviewed (approved):
LGTM, once you add the top-level grafana dashboards, as discussed offline.

Commit:
Add grafana dashboards. Fix typo

mberhault commented:
Fixed typo and added grafana dashboards. We may want to give a bit of info about each:
* runtime: node status: uptime, memory, cpu, ...
* storage: storage availability
* sql: sql queries/transactions
* replicas: replica information and replica operations

jseldess commented:
Thanks, @mberhault. Still LGTM.
